### PR TITLE
SHOW statements docs (part I)

### PR DESCRIPTION
**Pull request by randomJoe211**
Most of the content is migrated from v1.0. But all the statements have been verified and their examples have been updated. And there's some minor optimization.

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/2.show-collation.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# SHOW COLLATION
+
+The `SHOW COLLATION` statement shows the collations supported by Nebula Graph.
+
+Currently available types are: utf8_bin, utf8_general_ci, utf8mb4_bin and utf8mb4_general_ci. When the character set is utf8, the default collate is utf8_bin; when the character set is utf8mb4, the default collate is utf8mb4_bin. Both utf8_general_ci and utf8mb4_general_ci are case-insensitive.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/2.show-collation.md:5]: utf8mb4_bin, and utf8mb4_general_ci.

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# SHOW CONFIGS
+
+The `SHOW CONFIGS` statement lists the mutable configurations of the Graph Service, Meta Service or Storage Service.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md:3]: Meta Service, or Storage Service.

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+For how to update the configurations nGQL, see CONFIGS syntax (doc TODO).
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md:5]: the configurations nGQL?

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/4.show-create-space.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# SHOW CREATE SPACE
+
+The `SHOW CREATE SPACE` statement shows the basic information of the specified graph space, such as the nGQL for creating the graph space, the partition number, the replica number and so on.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/4.show-create-space.md:3]: remove " and so on". If "such as" is used, "and so on" is not necessary.

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+| "GRAPH" | "slow_op_threshhold_ms"   | "int" | "MUTABLE" | 50    |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "heartbeat_interval_secs" | "int" | "MUTABLE" | 3     |
++---------+---------------------------+-------+-----------+-------+
+| "GRAPH" | "meta_client_retry_times" | "int" | "MUTABLE" | 3     |
++---------+---------------------------+-------+-----------+-------+
+Got 6 rows (time spent 1216/1880 us)
+```
+
+The output of `SHOW CHARSET` is explained as follows:
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md:33]: SHOW CONFIG

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# SHOW CONFIGS
+
+The `SHOW CONFIGS` statement lists the mutable configurations of the Graph Service, Meta Service or Storage Service.
+
+For how to update the configurations nGQL, see CONFIGS syntax (doc TODO).
+
+## Syntax
+
+```ngql
+SHOW CONFIGS [graph|meta|storage]
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md:10]: if no service name is specified, all the configurations are returned

Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/3.show-configs.md:10]: My fault. I think `graph|meta|storage` are keywords so pls capitalize them.

**Review by whitewum — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**